src/Settings/Settings.ts

```diff
@@ -192,6 +192,8 @@ export const Settings = {
   useEngineeringNotation: false,
   /** Whether to disable suffixes and always use exponential form (scientific or engineering). */
   disableSuffixes: false,
+  /** The default amount of digits displayed after the period '.' in a number. */
```

Comment:
Optional suggestion:
```suggestion
  /** The default amount of digits displayed after the decimal separator. */
```
"Decimal point" and "decimal comma" are more common terms, but their usage depends on countries.